nextjs_server/src/app/mockData/calendarEvent.json

```diff
@@ -1,221 +1,241 @@
 [
   {
-    "id": "1",
+    "id": 1,
```

Comment:
Do we still need this file?

Reply:
@SpencerKing7 No we shouldn't need it anymore